staff/pablo-rey/precourse/pasapalabra-pro/css/main.scss

```diff
@@ -0,0 +1,115 @@
+$donut-diameter: 500px;
+$player-diameter: $donut-diameter + 100px;
+$letter-diameter: 50px;
+$num-letters: 28;
+$start-angle: -90deg - (360deg / $num-letters) / 2 ;
+
+$primary: #2780E3;
+$secondary: #373a3c;
+$success:#3FB618;
+$info:#9954BB;
+$warning:#FF7518;
+$danger:#FF0039;
+$light:#f8f9fa;
+$dark:#373a3c;
+
+.player {
+  width: $player-diameter;
+  height: $player-diameter;
+}
+
+.player header {
+  font-size: 3rem;
+  text-align: center;
+
+  position: absolute;
+  width: $player-diameter * 0.5;
+  background-color: $light;
+  align-content: center;
+  border-radius: 15px;
+  border: 2px solid $secondary;
+  transform: translate($player-diameter * 0.25, $player-diameter * .25) ;
+}
+
+.timer {
+  font-size: 2rem;
+  text-align: center;
+  position: absolute;
+
+  width: $player-diameter * 0.3;
+  background-color: $light;
+  border-radius: 15px;
+  border: 2px solid $secondary;
+  transform: translate($player-diameter * .35, $player-diameter * .4) ;
+}
+
+.donut {
+  width: $donut-diameter;
+  height: $donut-diameter;
+  position: absolute;
+}
+
+.letter {
+  width: $letter-diameter;
+  height: $letter-diameter;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+
+  border: 2px solid $secondary;
+  border-radius: 50%;
+  background-color: $primary;
+
+  color: white;
+  text-align: center;
+  text-justify: center;
+  justify-content: center;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: 600;
+  font-size: 2rem;
+  margin: $letter-diameter / 2;  
+}
+
+@for $i from 1 to $num-letters {
+  .donut div:nth-child(#{$i}) {
+    transform: rotate($start-angle + $i * 360deg / ($num-letters - 1)) translate($donut-diameter / 2) translate(0px) rotate(-($start-angle + $i * 360deg / ($num-letters - 1))); 
+    animation-name: letterActive; 
+    animation-duration: 2s; 
+    animation-iteration-count: 0;   
+  }
+}
+
+@keyframes letterActive {
+  0% { background-color: $primary; }
+  50% { background-color: $light; color:$dark; }
+  100% { background-color: $primary; }
+}
+
+h1 {
+  text-align: center;
+}
+
+#question-letter-circle {
+  width: 6rem;
+  height: 6rem;
+  margin: 10px auto;
+  border: 3px solid $secondary;
+  border-radius: 50%;
+  text-align: center;
+  transform: translateX(30px)
```

Comment:
@paulusrex even that this code works is better to not forget to end css lines with ;

Reply:
Thanks. The linter usually makes this kind of work, but I don't know why the linter doesn't correcct this lines.

Reply:
Usually the linter put all semicolons in the code. Thanks for the comment. I will save the entire file lintering.